tests/models/test_models_vae.py

```diff
@@ -270,14 +270,6 @@ def tearDown(self):
         gc.collect()
         torch.cuda.empty_cache()
 
-    def get_file_format(self, seed, shape):
-        return f"gaussian_noise_s={seed}_shape={'_'.join([str(s) for s in shape])}.npy"
-
-    def get_sd_image(self, seed=0, shape=(4, 3, 512, 512), fp16=False):
-        dtype = torch.float16 if fp16 else torch.float32
-        image = torch.from_numpy(load_hf_numpy(self.get_file_format(seed, shape))).to(torch_device).to(dtype)
-        return image
```

Comment:
Could I get an explanation on why is this going away?

Reply:
Thanks for reviewing! I had removed these because they weren't needed for the new test - but re-added them now.